erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -9,73 +9,89 @@
 from frappe.model.document import Document
 from frappe.model.mapper import get_mapped_doc
 from frappe.utils import (
-	cint,
-	date_diff,
-	flt,
-	get_datetime,
-	get_link_to_form,
```

Comment:
Please undo all formatting changes. Disable auto-formatter if you have any 😅

Reply:
https://github.com/frappe/erpnext/wiki/Preparing-a-Contribution-Proposal#note-about-formattingstyle-changes